client/jmx-adapter/src/main/java/org/jolokia/client/jmxadapter/ToOpenTypeConverter.java

```diff
@@ -201,6 +201,9 @@ public static OpenType<?> recursivelyBuildOpenType(String name, Object rawValue)
         return new CompositeType("complex", "complex", keys, keys, types);
       }
     }
+    if( rawValue == null ) {
+      return VOID;
```

Comment:
This one is up for discussion. For Java Mission Control at least it seems that this is a better solution than triggering a runtime exception, even though we fail in retrieving the type. 
![3896AE0E-44DE-414C-ACE4-E5B3AA3794A6](https://user-images.githubusercontent.com/6298906/166158666-9c9e5ed7-570f-4b07-a9ae-2d81c8d6cbde.jpeg)
^Example of a case where it works out ok.